

In file included from arch/ppc64/kernel/signal.c:29:
include/asm/ppc32.h:144: redefinition of `struct compat_sigevent'
include/asm/ppc32.h:156: redefinition of `compat_sigevent_t'
include/linux/compat.h:107: `compat_sigevent_t' previously declared here


---

 25-power4-akpm/include/asm-ppc64/ppc32.h |   14 --------------
 1 files changed, 14 deletions(-)

diff -puN include/asm-ppc64/ppc32.h~compat_mq-ppc-fix include/asm-ppc64/ppc32.h
--- 25-power4/include/asm-ppc64/ppc32.h~compat_mq-ppc-fix	2004-04-08 13:21:48.262814248 -0700
+++ 25-power4-akpm/include/asm-ppc64/ppc32.h	2004-04-08 13:22:35.262669184 -0700
@@ -141,20 +141,6 @@ struct ucontext32 { 
 	struct mcontext32	uc_mcontext;
 };
 
-typedef struct compat_sigevent {
-	compat_sigval_t sigev_value;
-	int sigev_signo;
-	int sigev_notify;
-	union {
-		int _pad[SIGEV_PAD_SIZE];
-		int _tid;
-		struct {
-			compat_uptr_t _function;
-			compat_uptr_t _attribute;
-		} _sigev_thread;
-	} _sigev_un;
-} compat_sigevent_t;
-
 struct ipc_kludge_32 {
 	unsigned int msgp;
 	int msgtyp;

_
